ui/package.json

```diff
@@ -27,7 +27,7 @@
     "@angular/platform-browser": "~5.1.0",
     "@angular/platform-browser-dynamic": "~5.1.0",
     "@angular/router": "~5.1.0",
-    "@ngui/auto-complete": "^0.13.2",
+    "@ngui/auto-complete": "https://github.com/bnjjj/auto-complete",
```

Comment:
Make a npm release of this ( in case of you'll remove your repository :in the future